protoactor/protos_pb2.py

```diff
@@ -0,0 +1,267 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
It's a normal practice sometimes to include autogenerated files which are used in the code into the repo. For example .Net implementation repo also contains autogenerated files. https://github.com/AsynkronIT/protoactor-dotnet/blob/dev/src/Proto.Actor/Protos.g.cs